services.go

```diff
@@ -897,7 +897,7 @@ type SlackService struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/services.html#slack
 type SlackServiceProperties struct {
-	WebHook                   string    `json:"web_hook"`
+	WebHook                   string    `json:"webhook"`
```

Comment:
```suggestion
	Webhook                   string    `json:"webhook"`
```

Reply:
This is a breaking change

Reply:
Interesting... I now see they are all "wrong" in this file. I guess that is also why this slipped in. I used the add tags logic from `gopls` which converts CamelCase to snake_case to create the tag names. But in this case (while it's clearly wrong) I'm fine with leaving it like this...

Reply:
Yeah I'd obviously prefer not renaming the struct field so we don't need to deal with the breaking change. 😄 

Though if you have code generation working I can see why you'd rename it.